GAE/src/org/waterforpeople/mapping/app/web/rest/QuestionRestService.java

```diff
@@ -448,4 +420,21 @@ private Question newQuestion(QuestionDto dto) {
         final Question result = questionDao.save(q);
         return result;
     }
+
+    private void attachAnyOptions(QuestionDto dto, List<QuestionOptionDto> qoResults) {
```

Comment:
As a proposal, since the `qoResults` is never used locally besides adding elements to the list, perhaps we could eliminate it from the parameter list and instead have a return type `List<QuestionOptionDto>`